pkg/network/infraconfigurators/bridge_test.go

```diff
@@ -701,6 +843,38 @@ var _ = Describe("Bridge infrastructure configurator", func() {
 					mac, podIP).GenerateNonRecoverableDHCPConfig()).To(Equal(&expectedDhcpConfig))
 			})
 		})
+
+		When("IPAM is enabled with an IPv6 address", func() {
+			var (
+				mac   net.HardwareAddr
+				podIP netlink.Addr
+			)
+
+			createBridgeConfiguratorWithIPAM := func(mac net.HardwareAddr, podIP netlink.Addr, routes ...netlink.Route) *BridgePodNetworkConfigurator {
+				bc := NewBridgePodNetworkConfigurator(vmi, iface, bridgeIfaceName, launcherPID, handler)
+				bc.podNicLink = &netlink.GenericLink{LinkAttrs: netlink.LinkAttrs{Name: ifaceName}}
+				bc.vmMac = &mac
+				bc.ipamEnabled = true
+				bc.podIfaceIPv6 = &podIP
+				bc.podIfaceRoutes = routes
+				return bc
+			}
+
+			BeforeEach(func() {
+				podIP = netlink.Addr{IPNet: &net.IPNet{IP: net.IP{0xfd, 0x23, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0x6}, Mask: net.CIDRMask(64, 128)}}
+				mac, _ = net.ParseMAC("AF:B3:1F:78:2A:CA")
+			})
+
+			It("generates a DHCP config with MAC / IP address information", func() {
+				expectedDhcpConfig := cache.DHCPConfig{
+					IPAMDisabled: false,
+					MAC:          mac,
+					IPv6:         podIP,
+				}
+				Expect(createBridgeConfiguratorWithIPAM(
+					mac, podIP).GenerateNonRecoverableDHCPConfig()).To(Equal(&expectedDhcpConfig))
```

Comment:
nit: I would prefer this to be:
```golang
Expect(
    createBridgeConfiguratorWithIPAM(mac, podIP).GenerateNonRecoverableDHCPConfig(),
).To(Equal(&expectedDhcpConfig))
```

This is just aesthetic, feel free to disregard my comment.